### ISSUE TITLE
Allow `Create test file` to Accept Folders from `AssemblyMetadata`

### ISSUE THREAD

**Issue by Mike-E-angelo** [zvirja/ReSharperHelpers#14]
To start with, I am SO very happy with your `Create test file` action!  It is something I have been asking/searching for some time now.  Before TestLinker was archived, it was an active request:
https://github.com/matkoch/resharper-testlinker/issues/10#issuecomment-633994792

It even works with files with multiple classes in it.  So if I am actively developing in a class file with multiple classes in it (happens quite often in my development cycle), it detects the correct class and creates the correct file in the correct location.

THANK YOU. 🎉

I have everything exactly as I'd like it with the exception of one item, and it sort of goes back to the TestLinker discussion above.

What I would like to see is enabling the `AssemblyMetadata` to accept folders.

So if I have an application project `Application.Feature.CoolFeature` and a testing project `Testing`, I could apply this attribute to `Application.Feature.CoolFeature`:

```
[assembly: AssemblyMetadata("ReSharperHelpers.TestProject", "Testing/Features/CoolFeature")]
```

And if I hit `Create Test File` on a class `NeatComponent` in `Application.Feature.CoolFeature/Namespace/NeatComponent.cs`, it would create:

```
Features/CoolFeature/Namespace/NeatComponentTests.cs
```

In the `Testing` project.

Possible? :) :) :)

Thank you for any consideration and your great work here. 👍

**Comment by zvirja** [comment]
I've just closed #9 which looks similar. Could you please review whether it covers the current issue as well? If not, please describe the scenario, so I'll look what could be done 😊

**Comment by zvirja** [comment]
You are welcome! Thanks for the warm words 😊 

Yes. It should take the relative namespace of your test class and create `<testing proj>/Feature/CoolFeature/<relative namespace>/<class>Tests`. I haven't tested it deeply, so feel free to come back if there are some issues with it 😉
